priv/json.erl.eex

```diff
@@ -34,19 +34,13 @@ request(Client, Action, Input, Options) ->
     Client1 = Client#{service => <<"<%= context.signing_name %>">>},
     Host = get_host(<<"<%= context.endpoint_prefix %>">>, Client1),
     URL = get_url(Host, Client1),
-    Headers1 =
-        case maps:get(token, Client1, undefined) of
-            Token when byte_size(Token) > 0 -> [{<<"X-Amz-Security-Token">>, Token}];
-            _ -> []
-        end,
-    Headers2 = [
+    Headers1 = [
         {<<"Host">>, Host},
         {<<"Content-Type">>, <<"application/x-amz-json-<%= context.json_version %>">>},
         {<<"X-Amz-Target">>, << <<"<%= context.target_prefix %>.">>/binary, Action/binary>>}
-        | Headers1
     ],
     Payload = jsx:encode(Input),
-    Headers = aws_request:sign_request(Client1, <<"POST">>, URL, Headers2, Payload),
+    Headers = aws_request:sign_request(Client1, <<"POST">>, URL, Headers1, Payload),
```

Comment:
Super minor nit, but you might consider `s/Headers1/Headers/` and `s/Headers/SignedHeaders/` for a touch more clarity.

Reply:
I can definitely make that change. 😄